docs/workloads/azure/data/etl_pipelines/etl_intro_data_azure.md

```diff
@@ -0,0 +1,45 @@
+---
+id: etl_intro_data_azure
+title: ETL Overview
+sidebar_label: ETL Overview
+hide_title: false
+hide_table_of_contents: false
+description: Introduction to ETL Pipelines
+keywords:
+  - ingest
+  - silver
+  - gold
+  - bronze
+  - databricks
+  - etl
+  - medallion
+---
+
+Stacks Data Platform showcases an array of integrated data processing pipelines, designed to demonstrate its full
+spectrum of capabilities. Our goal is to offer a comprehensive user journey that begins at the data source and travels
+through multiple transformations within the data lake layers, culminating in visualizations on graphical dashboards.
+
+## Data pipelines
+
+The naming convention for the data pipelines originates in Databricks' Medallion Architecture, which emphasises
+structured data transformation layers:
+
+* [Data Ingestion](ingest_data_azure.md): The initial stage of data collection.
+* [Silver Pipeline](silver_data_azure.md): Intermediate processing and transformation stage.
+* Gold Pipeline: The refined and processed data ready for analytics.
+
+### ETL utilities
+
+This section also introduces several auxiliary utilities to complement the main data pipelines:
+
+* [Data Quality](data_quality_azure.md): An optional step incorporated into all ETL pipelines to ensure data integrity.
+* [Datastacks](datastacks.md): A CLI tool designed to facilitate the creation of custom pipelines using predefined templates.
+* [PySparkle](../../../common/data/pysparkle/pysparkle_quickstart.md): A versatile library equipped with reusable data processing components.
+
+## Dataset
+
+The platform leverages [The Movies Dataset](https://www.kaggle.com/datasets/rounakbanik/the-movies-dataset) from Kaggle,
```

Comment:
Here I would just reference and link to the new page I added about the example dataset (once merged in).

Reply:
OK, I'll wait for yours to get merged first, and maybe will add in there some additional info about this dataset (since I removed it from other places) and then reference it here.

Reply:
Reference updated, but I left this small description - let me know if you're OK with that.